velox/common/file/FileSystems.h

```diff
@@ -43,6 +43,14 @@ struct FileOptions {
 
   std::unordered_map<std::string, std::string> values;
   memory::MemoryPool* pool{nullptr};
+  std::optional<int64_t> fileSize{std::nullopt};
+
+  int64_t getFileSize() const {
```

Comment:
`std::optional<int64_t> getFileSize()` Let's return the optional value directly here.